client/src/components/Toasts/Toast.js

```diff
@@ -114,14 +126,21 @@ const NotificationToast = () => {
       }}
       message={
         <>
-          <div className="notification-toast-icon-wrapper">
-            <i className="icon icon-history" />
-          </div>
-          <div className="notification-toast-content">
-            <span>
-              {notification ? getToastMessage(notification) : undefined}
-            </span>
-          </div>
+          {notification && (
```

Comment:
Now that the html in the `message` is more extensive, can it be extracted into its own component? This would also move the `getToastMessage` utility out of `NotificationToast` and simplify it.